crates/matrix-sdk/src/room/timeline/event_item.rs

```diff
@@ -490,58 +489,33 @@ impl From<RoomEncryptedEventContent> for EncryptedMessage {
     }
 }
 
-#[derive(Clone, Debug)]
-pub struct BundledReactions {
-    /// Whether all reactions are known, or some may be missing.
-    ///
-    /// If this is `false`, the remaining reactions can be fetched via **TODO**.
-    pub complete: bool, // FIXME: Unclear whether this is needed
-
-    /// The reactions.
-    ///
-    /// Key: The reaction, usually an emoji.\
-    /// Value: The count.
-    pub bundled: IndexMap<String, ReactionDetails>,
-}
-
-impl From<Box<AnnotationChunk>> for BundledReactions {
-    fn from(ann: Box<AnnotationChunk>) -> Self {
-        let bundled = ann
-            .chunk
-            .into_iter()
-            .filter_map(|a| {
-                (a.annotation_type == AnnotationType::Reaction).then(|| {
-                    let details =
-                        ReactionDetails { count: a.count, senders: TimelineDetails::Unavailable };
-                    (a.key, details)
-                })
-            })
-            .collect();
+/// The reactions grouped by key.
+///
+/// Key: The reaction, usually an emoji.\
```

Comment:
```suggestion
/// Key: The reaction, usually an emoji.
///
```

?

Reply:
If you want, it's just that with this syntax it doesn't create a new paragraph between the sentences, it's just a line break.